content/en/opentelemetry/interoperability/environment_variable_support.md

```diff
@@ -0,0 +1,153 @@
+---
+title: Using OpenTelemetry Environment Variables with Datadog SDKs
+further_reading:
+    - link: '/tracing/trace_collection/library_config/dotnet-core'
+      tag: 'Documentation'
+      text: '.NET Core SDK Configuration'
+    - link: '/tracing/trace_collection/library_config/dotnet-framework'
+      tag: 'Documentation'
+      text: '.NET Framework SDK Configuration'
+    - link: '/tracing/trace_collection/library_config/go'
+      tag: 'Documentation'
+      text: 'Go SDK Configuration'
+    - link: '/tracing/trace_collection/library_config/java'
+      tag: 'Documentation'
+      text: 'Java SDK Configuration'
+    - link: '/tracing/trace_collection/library_config/nodejs'
+      tag: 'Documentation'
+      text: 'Node.js SDK Configuration'
+    - link: '/tracing/trace_collection/library_config/php'
+      tag: 'Documentation'
+      text: 'PHP SDK Configuration'
+    - link: '/tracing/trace_collection/library_config/python'
+      tag: 'Documentation'
+      text: 'Python SDK Configuration'
+    - link: '/tracing/trace_collection/library_config/ruby'
+      tag: 'Documentation'
+      text: 'Ruby SDK Configuration'
+---
+
+After you set up the SDK library with your code and configure the Agent to collect APM data, optionally configure the SDK library as desired using OpenTelemetry environment variables
```

Comment:
An addition I think would be really useful to start this page is a _why_. If someone clicks on `OpenTelemetry > Interopability with Datadog > Environment Variable Configuration` they might still wonder if this page applies to them. I think a couple of sentences to reiterate this use case would be helpful, and I think @tushar-datadog can advise. Off the top of my head, I was thinking something like:

```
Since the Datadog SDKs implement the OpenTelemetry Tracing APIs and are continually being updated to further implement additional OpenTelemetry APIs, you can replace the OpenTelemetry SDK in your application with the Datadog SDK to receive traces and additional Datadog-supported signals with minimal changes to SDK configuration when using the OpenTelemetry SDK configurations outline below.
```

Reply:
@brett0000FF I wonder if you have any opinions on this suggestion? If you don't think we need to change anything then feel free to hit `Resolve conversation` 👍🏼

Reply:
@zacharycmontoya - Good question. I missed this suggestion of yours. I agree that it'd be helpful to include a bit more detail about the use cases at the top of the page. We could create a small Overview section at the top to expand on these details. @khanayan123 , let me know if you'd like me to commit an overview section to this PR. 👍